cypress/support/pages/admin/pageAdministration.js

```diff
@@ -1008,12 +1008,17 @@ class Management {
 		);
 	}
 
-	seeCustomParameterFormContains(paramName, value) {
-		cy.get(`[data-testid="${paramName}"]`).find("input").should("have.value", value);
+	seeCustomParameterFormContains(value) {
```

Comment:
method name is not correct. should be something like `verifyCustomSchoolParameterValue`